lib/common/db-renderable-content.js

```diff
@@ -191,7 +191,7 @@ function dbRenderableContentServiceFactory(
             i += 1;
         });
 
-        return waterline[self.collectionName].find({name: name})
+        return waterline[self.collectionName].find({name: name, scope: scope})
```

Comment:
Since Cassandra adapter doesn't support combined search, I changed to use a filter for designated scope after waterline.find(name) returns. @zyoung51